tests/expressions_window/tests.py

```diff
@@ -536,7 +536,7 @@ def test_multiple_ordering(self):
             ('Brown', 53000, 'Sales', datetime.date(2009, 9, 1), 108000),
         ], transform=lambda row: (row.name, row.salary, row.department, row.hire_date, row.sum))
 
-    @skipIf(connection.vendor == 'postgresql', 'n following/preceding not supported by PostgreSQL')
```

Comment:
This restriction was lifted in PostgreSQL 11.

See https://modern-sql.com/blog/2019-01/sqlite-in-2018#window-functions

Reply:
I know but the test fails on PostgreSQL 11 somehow and I didn't want to investigate more

Using django-docker-box

```
$ POSTGRES_VERSION=11 docker-compose run --rm postgres --noinput expressions_window
[snip]
AssertionError: 'PostgreSQL only supports UNBOUNDED together with PRECEDING and FOLLOWING.' not found in 'RANGE with offset PRECEDING/FOLLOWING is not supported for column type date and offset type integer\nLINE 1: ...e_date" ASC RANGE BETWEEN UNBOUNDED PRECEDING AND 1 FOLLOWIN...\n                                                             ^\nHINT:  Cast the offset value to an appropriate type.\n'
```

It looks like there might be a bug in PostgreSQL 11 actually.

Anyway, this should be handled in a different PR.